pkg/broker/handler/processors/filter/processor.go

```diff
@@ -55,10 +57,24 @@ func (p *Processor) Process(ctx context.Context, event *event.Event) error {
 		return nil
 	}
 
+	trigger := types.NamespacedName{
+		Namespace: target.Namespace,
+		Name:      target.Name,
+	}
+	var span *trace.Span
```

Comment:
nit: might as well wrap this into a local func so that here it will look like `ctx, span := startSpan(ctx, target, event); defer span.End()`